mmv1/third_party/terraform/services/securitycentermanagement/resource_scc_management_organization_event_threat_detection_custom_module_test.go.erb

```diff
@@ -112,6 +119,7 @@ resource "google_scc_management_organization_event_threat_detection_custom_modul
 			  ]}
             EOF
 }
+<% end -%>
```

Comment:
`<% end -%>` needs to be moved to the bottom.

Reply:
Addressed.